skimage/measure/tests/test_regionprops.py

```diff
@@ -704,6 +704,13 @@ def test_intensity_mean():
     assert_almost_equal(intensity, 1.02777777777777)
 
 
+def test_intensity_median():
+    intensity = regionprops(SAMPLE, intensity_image=INTENSITY_SAMPLE)[
+        0
+    ].intensity_median
+    assert_almost_equal(intensity, 1.0)
```

Comment:
Perhaps test on a more meaningful input image since `SAMPLE` only contains 0 and 1?

Reply:
`INTENSITY_SAMPLE` at least has `0`, `1`, and `2`, so that `intensity_mean` is `1.02777` and `intensity_median` is `1.0`.

Reply:
Ah, I see. Cary on then. :smile: